compass/landice/tests/calving_dt_convergence/dt_convergence_test/__init__.py

```diff
@@ -0,0 +1,109 @@
+from compass.testcase import TestCase
+from compass.landice.tests.calving_dt_convergence.run_model import RunModel
+# from compass.validate import compare_variables  # not currently used
+import numpy
+import netCDF4
+import matplotlib.pyplot as plt
+import matplotlib.cm
+
+
+class DtConvergenceTest(TestCase):
+    """
+    A test case for running the same configuration with a series of
+    values for config_adaptive_timestep_calvingCFL_fraction
+    to check for convergence
+    """
+
+    def __init__(self, test_group, mesh, calving, velo):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.landice.tests.mismipplus.MISMIPplus
+            The test group that this test case belongs to
+
+        """
+        name = f'calving_dt_convergence_test_{mesh}_{calving}_{velo}'
+        subdir = f'{mesh}.{calving}.{velo}'
+        super().__init__(test_group=test_group, name=name, subdir=subdir)
+
+        cores = 36
+        min_cores = 4
+
+        # Do fewer runs if FO solver
+        if velo == 'FO':
+            self.fractions = numpy.arange(0.25, 2.3, 0.25)
+        else:
+            self.fractions = numpy.arange(0.2, 3.1, 0.2)
+
+        for frac in self.fractions:
+            name = f'frac{frac:.2f}'
+            step = RunModel(test_case=self, name=name,
+                            mesh=mesh,
+                            calving=calving,
+                            velo=velo,
+                            calv_dt_frac=frac,
+                            cores=cores, min_cores=min_cores, threads=1)
+            self.add_step(step)
+
+    # no configure() method is needed
+
+    # no run() method is needed
+
+    def validate(self):
+        """
+        Test cases can override this method to perform validation of variables
+        and timers
+        """
+        # variables = ['thickness', 'surfaceSpeed']
```

Comment:
Maybe it would be good to add a note here that if you want to use compare_variables, you should uncomment line 3 as well